jar-connector/src/main/java/com/sixsq/slipstream/connector/SystemConfigurationParametersFactoryBase.java

```diff
@@ -34,6 +33,20 @@
 
 	protected Map<String, ServiceConfigurationParameter> referenceParameters = new HashMap<String, ServiceConfigurationParameter>();
 
+	public static final String NATIVE_CONTEXTUALIZATION_KEY = "native-contextualization";
+
+	public static final String NATIVE_CONTEXTUALIZATION_NEVER = "never";
+	public static final String NATIVE_CONTEXTUALIZATION_ALWAYS = "always";
+	public static final String NATIVE_CONTEXTUALIZATION_LINUX_ONLY = "linux only";
+	public static final String NATIVE_CONTEXTUALIZATION_WINDOWS_ONLY = "windows only";
```

Comment:
Spaces before `only` scare me (although I know that all the params are wrapped into the single quotes later).  Can we add dash or underscore please?